WordPress/Classes/ViewRelated/Pages/PageListViewController.swift

```diff
@@ -356,6 +360,16 @@ class PageListViewController: AbstractPostListViewController, UIViewControllerRe
         })
     }
 
+    func presentAlertForPageBeingUploaded() {
+        let message = NSLocalizedString("Page is being uploaded, please wait until this is done", comment: "Prompts the user that the post is being uploaded and cannot be restored post was moved to the drafts list.")
```

Comment:
I'm not sure the second bit of the comment belongs here - "post was moved to the drafts list"
I also wonder if we want to run the text past editorial. Something more like "Page currently uploading. You must wait for it to finish before editing."? (I still don't think that's right though)

Same comment for the Posts alert too!

Reply:
Humm, I can pass the text to them.